docs/finalproject/week-1/lesson.md

```diff
@@ -22,41 +23,55 @@ sidebar_label: Lesson
 
 Once you have your team and your assignment, start by sorting out the basic administrative requirements.
 
-```txt
-1. Give your team a name!
-   - This will be your identity
-   - Use it in all communications/presentations.
-```
+### 1. Introductions
+   - Before we introduce ourselves let's introduce one of the tools we'll be using to collaborate during the project, [Miro](https://miro.com/). Whiteboards are a simple yet powerful tool software teams use to share ideas and solve problems - Miro is just a virtual version. It should be intuitive but if you want help there's lots of [content online](https://www.youtube.com/watch?v=pULLAEmhSho).
+   
+   - Once you're familiar with the tool one person should make a copy of [this Miro board](https://miro.com/app/board/o9J_knoA0Ag=/) for your team to work on. 
+   
+   - OK, now we're setup with let's get to know each other a little using **Exercise 1: Icebreaker** on your team's copy of the Miro board.
 
-```txt
-2. Create a public Slack channel
+
+### 2. Give your team a name!
```

Comment:
Looks like the teams have already mostly got their names, they're mostly in the sign up Google Sheet